spec/services/properties_creation_service_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'rails_helper'
+
+RSpec.describe PropertiesCreationService do
+  before { stub_call_to_json_schema }
+
+  let(:assessment) { create :assessment }
+
+  shared_examples 'error response' do
+    it 'does not create any property records' do
+      expect {
+        described_class.call(payload)
```

Comment:
To follow RSpec's convention, you can define `subject` at the top (just below `let(:assessment)`) and then always use `subject`

```ruby
subject { described_class.call(payload) }
...
...
expect { subject }.not_to change { Property.count }
```